docs/transactions/iti41.md

```diff
@@ -1,35 +1,258 @@
+
+### Referencing external documents
+When publishing CH-EMED-EPR documents, all references (from the key elements) shall be known to the eMedication service. All non-key elements are ignored by the eMedication service. 
 
-When publishing a CH-EMED-EPR documents, all references (from the key elements) shall be known to the eMedication service.
-All non-key elements are ignored by the eMedication service.
+This implies that the metadata cannot contain [Reference ids](https://profiles.ihe.net/ITI/TF/Volume3/ch-4.2.html#4.2.3.2.28). This option is not supported, and submission of a reference to an existing document is forbidden.
 
-Some values from the DocumentEntry are checked against the document:
+### Creation times
+[Creation time](https://build.fhir.org/ig/CARA-ch/ch-emed-epr/StructureDefinition-ch-emed-epr-composition-medicationtreatmentplan-definitions.html#Composition.date) of a document must be equal or greater than the creation time of the last document of the treatment chain, ie. the last published document targeting the treatment (either the original MTP if no other document, or the last document published referencing this treatment).
+* Creation time of the document has to be in the past.
+* All references to other documents or items have to refer to existing approved non deleted documents.
+
+### Rules for each type of document
+#### PRE
+* Each PRE item should [refer to an MTP item](https://build.fhir.org/ig/CARA-ch/ch-emed-epr/StructureDefinition-ch-emed-epr-document-medicationprescription.html) (`CHEMEDEPRMedicationRequest.treatmentplan`) that must:
+    * Exist (has already been published, and never deleted)
+    * Be [approved](https://profiles.ihe.net/ITI/TF/Volume3/ch-4.2.html#4.2.3.2.2), ie. `DocumentEntry.availabilityStatus = approved`, 
+    * Preferably  be valid, ie. the submission date should be within the [MTP `dosage`'s](https://build.fhir.org/ig/CARA-ch/ch-emed-epr/StructureDefinition-ch-emed-epr-dosage.html) `boundsPeriod` (`CHEMEDEPRDosage.repeat.bounds`).
+        * If the `boundsPeriod` is not defined, the entry is considered to be always active.
+        * If only a [`startDate`](http://hl7.org/fhir/R4/datatypes-definitions.html#Period.start) is specified, the MTP is considered to be active if the current date is after that date.
+    * Active, ie. `CHEMEDEPRDocumentMedicationTreatmentPlan.CHEMEDEPRMedicationStatement.status = active`
+    * For the same patient.
+* Code of [medication](https://build.fhir.org/ig/CARA-ch/ch-emed-epr/StructureDefinition-ch-emed-epr-medication.html) should match the code of medication of the [MTP referenced by the PRE](https://build.fhir.org/ig/CARA-ch/ch-emed-epr/StructureDefinition-ch-emed-epr-document-medicationprescription.html) (`CHEMEDEPRMedicationRequest.treatmentplan`). _This rule is not enforced and might be extended in the future_.    
```

Comment:
@dvribeira I'm not sure about substitution here : does it not occur at DIS level ? Shouldn't PRE always be 100% aligned with MTP ?

Reply:
- Yes, substitution should not be done at the PRE level and in fact it cannot flagged. Having said that, what should we do if a prescription does not provide the same medication as the MTP?
  - Should we reject the PRE submission altogether?
  - Should we accept it with a warning and then:
    - Accept the medication of the PRE?
    - Ignore the medication of the PRE?
In my opinion the document should be accepted along with the specified medication but with a warning being returned upon submission. Having said that, this can be discussed with everybody and more particularly with professionals.